test/validators.js

```diff
@@ -1,4 +1,4 @@
-var validator = require('../index'),
+﻿var validator = require('../index'),
```

Comment:
Can you remove the Unicode BOM you added here? See https://travis-ci.org/chriso/validator.js/jobs/292413822